.github/workflows/wheels.yml

```diff
@@ -0,0 +1,53 @@
+name: RMM wheels
+
+on:
+  push:
+    branches:
+      - 'pull-request/[0-9]+'
+  workflow_call:
+    inputs:
+      versioneer-override:
+        type: string
+        default: ''
+      build-tag:
+        type: string
+        default: ''
+      branch:
+        required: true
+        type: string
+      date:
+        required: true
+        type: string
+      sha:
+        required: true
+        type: string
+      build-type:
+        type: string
+        default: nightly
+
+concurrency:
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
+
+jobs:
+  rmm-wheels:
+    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-manylinux.yml@main
+    with:
+      repo: rapidsai/rmm
+
+      build-type: ${{ inputs.build-type || 'pull-request' }}
```

Comment:
No need for `|| 'pull-request'` when we remove the `on: push` above, since this will not (for now) run as a pull-request build, but only nightly.